test/functional/services/dashboard/visualizations.ts

```diff
@@ -95,6 +98,8 @@ export class DashboardVisualizationsService extends FtrService {
       saveAsNew: false,
       redirectToOrigin: true,
     });
+    await this.header.waitUntilLoadingHasFinished();
+    await this.dashboard.waitForRenderComplete();
```

Comment:
Removed duplicated call to `waitForRenderComplete` https://github.com/elastic/kibana/blob/main/test/functional/apps/dashboard/group5/empty_dashboard.ts#L58

Reply:
Done in f646bb95848c25441f10c212678ecc062c771854